bin/akula.rs

```diff
@@ -221,29 +226,54 @@ fn main() -> anyhow::Result<()> {
                                 .unwrap();
 
                             let mut api = Methods::new();
-                            api.merge(
-                                EthApiServerImpl {
-                                    db: db.clone(),
-                                    call_gas_limit: 100_000_000,
-                                }
-                                .into_rpc(),
-                            )
-                            .unwrap();
-                            api.merge(NetApiServerImpl { network_id }.into_rpc())
-                                .unwrap();
-                            api.merge(ErigonApiServerImpl { db: db.clone() }.into_rpc())
+
+                            let api_options: HashSet<String> = HashSet::from_iter(
+                                opt.enable_api
+                                    .into_iter()
+                                    .map(|s| s.to_lowercase())
+                                    .collect::<Vec<String>>(),
+                            );
```

Comment:
```suggestion
                            let api_options = opt.enable_api
                                .into_iter()
                                .map(|s| s.to_lowercase())
                                .collect::<HashSet<String>>();
```

Reply:
Ahh yes, thanks for pointing this out. The way I have it is redundant. Updating it now.